app/client/components/cancel/cancellationReason.ts

```diff
@@ -14,6 +14,9 @@ export interface CancellationReason {
 }
 
 export type CancellationReasonId =
+  | "mma_time"
+  | "mma_better_offer"
+  | "mma_issue"
```

Comment:
New Zuora cancellation reason codes, add to Zuora in...
✅DEV
✅UAT
✅PROD

_Requested in https://docs.google.com/document/d/1hOeqMlKN54Vl4CDC8N2IJm_--K2KOKNJmNMkLRWuDvk_